server/src/callbacks.mjs

```diff
@@ -75,11 +76,17 @@ Empirica.onNewBatch(async function ({ batch }) {
     try {
       console.log("fetching");
       const fetched = await (await axios.get(url)).data;
+      try {
+        marked.parse(fetched);
+        console.log("Able to parse markdown");
```

Comment:
I don't think we need to log success, just failure.

Reply:
We may actually test for some other things as well as parsing: minimum length would be an easy way to avoid some simple failures. Let's check we have at least 75 non-whitespace characters, and if not throw an error.